internal_user/templates/init_issue_cheque.html

```diff
@@ -0,0 +1,52 @@
+{% extends "base.html" %}
+{% block javascript %}
+{% endblock %}
+{% block content %}
+
+<div>
+    <div>
+        <h2>Issue cashier cheque</h2><br><br>
+    </div>
+    <div class="form-group">
+        <form action='./searchCustomer' method="post">
+            <label>Search Customer:</label>
+            {% csrf_token %}
+            <input type="text" id="customerSearchString" class="form-control" name="customerSearchString" value={{customerSearchString}}><br>
+            <input type="hidden" name="context_page" value="issue_cheque">
+            <input type="submit" value="Search" class="btn btn-primary float-right app-submit"><br><br>
+        </form>
+    </div>
```

Comment:
Also here.. actually customer search per se is not asked for in the requirements. Although I totally agree it's needed in a real application, I was thinking it would be a good idea to keep it simple for want of time.

Reply:
Same as above